src/OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -21,9 +23,21 @@ namespace orbit_base {
 
 #ifdef _WIN32
 
+[[nodiscard]] inline uint64_t GetPerformanceCounterPeriodNs() {
```

Comment:
This also needs a small comment, especially now that it's in `OrbitBase`.

Reply:
Added comment.